oms-kemp-applicationdelivery-solution/azuredeploy.json

```diff
@@ -0,0 +1,906 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "omsLogAnalyticsWorkspaceName": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "Create new or refer to an existing OMS Log Analytics Workspace"
+            }
+        },
+        "omsLogAnalyticsRegion": {
+            "type": "string",
+            "defaultValue": "eastus",
+            "allowedValues": [
```

Comment:
yes, still in doubt if I should list the regions or not, since they're set to grow. anyway added AU.